config/database.yml

```diff
@@ -0,0 +1,29 @@
+# for postgres comment lines below in
+# It is preferable to use postgres because this
+# is what's used in production and it is necessary
+# to work on the coldmap.
+development: &development
+  adapter: postgresql
+  database: dev_database
+  username: dave_database_user
```

Comment:
For the development db, do you want to use the twinedb name ther ealso like twinedb_development or leave it is dev_database?